refactor/results/app/core/config.py

```diff
@@ -46,6 +46,10 @@ def assemble_cors_origins(cls, v: Union[str, List[str]]) -> Union[List[str], str
 
     class Config:
         case_sensitive = True
+        env_file = ".env"
 
 
 settings = Settings()
+
+print("Configuration:")
```

Comment:
are these debugging statements that should be removed?

Reply:
I've found in past projects there is a lot of value displaying app configuration when spinning up a service, so it is intended to stay there